frontend/apps/marketing/cicd/config.rb

```diff
@@ -48,7 +57,48 @@ module Configuration
       us-west-2
     ].freeze
 
+    # Unified site configuration using nested hash with deep merge approach
+    SITE_CONFIG = {
+      # Global defaults that apply to all site+environment combinations
+      defaults: {
+        contentful_env_id: 'master'
+      }.freeze,
+
+      # Site-specific configuration (applies to all environments for that site type)
+      site_defaults: {
+        corporate: {
+          contentful_space_id: '90t6bu6vlf76',
+        }.freeze,
+        csforall: {
+          contentful_space_id: '27jkibac934d',
+        }.freeze
+      }.freeze,
```

Comment:
I still think this should be an environment variable hash. Also the `CONTENTFUL_ENV_ID` should be one of the configurable environment variables as it may differ depending on the stack's use case.

Reply:
Yup - will implement that in the next Pull Request that updates the Marketing Sites template to actually provision a different site type.